automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -62,6 +64,46 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
         }
       }
 
+      "to create, clone, and delete v2 workspaces (using the Resource Buffer), in a v2 billing project" in {
```

Comment:
Is this a unit test or an integration test? Given that its in the automation folder, I assume integration?

Reply:
yep!